server/static/hardware_list.jsx

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const e = React.createElement;
+
+class HardwareList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { hardware: [] };
+    }
+
+    componentDidMount() {
+        this.update()
+        this.timerID = setInterval(
+            () => this.update(),
+            1000
+        );
+    }
+
+    update() {
```

Comment:
So this is setting the hardware state? Would it be better to have this just get the hardware state and we can maintain it in the flask app?